### DISCUSSION TITLE
Scrum Prep Milestone 3 - Week 4

### DISCUSSION THREAD

**Discussion by keisanabria** [uprm-inso4101-2024-2025-s1/semester-project-regiupr#197] [Scrum Preps]
Nov 18 - Nov 22

**Comment by keisanabria** [comment]
Got up to date with the project's communication channels. Asked Christian Marcial if he needed any help defining Issue #174 and which parts he would like to make progress on. Additionally, asked the Team Leader (Jorge) of Documentation if there were any lecture topic tasks to be assigned, as I would like to take them on. Requested to be assigned to Issues #181 and #187. Finally, was assigned to Issues #189 and #174.

**Comment by christianmedina-dev** [comment]
Registration Team. Worked on and finished Tasks (Issue #193) and (Issue #196).

**Comment by PedroRodz** [comment]
Not much besides opening two tasks.

**Comment by Gian-Go** [comment]
Published multiple documentation tasks. Developed a Petri Net Diagram for the documentation as well as work on small, miscleaneous changes within.

**Comment by cristianMarcial** [comment]
Multi Lang Support Team Leader Here || Modified the available screen modules so that they display a different language on their UI content strings according to the selected language.

**Comment by carlospepin23** [comment]
I worked on Lecture Topic Task #169, supported Task #172, and opened two additional Lecture Topic Tasks that are yet to be defined.

**Comment by nicolle-feliciano** [comment]
Started working on issues #188 and #190 of documentation

**Comment by Yamil-Serrano** [comment]
working on task #185 and #200

**Comment by Carloscaban** [comment]
Worked on registration team and assigned tasks
